modules/nodes-cluster-overcommit-qos-about.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-overcommit.adoc
+
+[id='nodes-cluster-overcommit-qos-about_{context}']
+= Understanding overcomitment and quality of service classes
+
+A node is _overcommitted_ when it has a pod scheduled that makes no request, or
+when the sum of limits across all pods on that node exceeds available machine
+capacity.
+
+In an overcommitted environment, it is possible that the pods on the node will
+attempt to use more compute resource than is available at any given point in
+time. When this occurs, the node must give priority to one pod over another. The
+facility used to make this decision is referred to as a Quality of Service (QoS)
+Class.
+
+For each compute resource, a container is divided into one of three QoS classes
+with decreasing order of priority:
+
+.Quality of Service Classes
+[options="header",cols="1,1,5"]
+|===
+|Priority |Class Name |Description
+
+|1 (highest)
+|*Guaranteed*
+|If limits and optionally requests are set (not equal to 0) for all resources
+and they are equal, then the container is classified as *Guaranteed*.
+
+|2
+|*Burstable*
+|If requests and optionally limits are set (not equal to 0) for all resources,
+and they are not equal, then the container is classified as *Burstable*.
+
+|3 (lowest)
+|*BestEffort*
+|If requests and limits are not set for any of the resources, then the container
+is classified as *BestEffort*.
+|===
+
+Memory is an incompressible resource, so in low memory situations, containers
+that have the lowest priority are terminated first:
+
+- *Guaranteed* containers are considered top priority, and are guaranteed to
```

Comment:
we need to expand on this at some point.  there are significant downsides to `Guaranteed` not limited to unneeded OOMKilling and savage latency (a cpu: 10m pod can experience 90ms of scheduling latency) due to CPU throttling.

Reply:
@sjenning @smarterclayton I will merge this PR and make changes for 4.0 in a separate PR, unless anyone objects.

Reply:
@sjenning @smarterclayton Any thoughts on what we want to document here? 

> there are significant downsides to Guaranteed